tck/features/expressions/literals/Literals7.feature

```diff
@@ -177,7 +177,7 @@ Feature: Literals7 - List
     Given any graph
     When executing query:
       """
-      RETURN [1, -2, 00, 71034856] AS literal
+      RETURN [1, -2, 0o77, 0xA4C, 71034856] AS literal
```

Comment:
You need to update the result assertion as well here.

Reply:
Splendid.